docs/tools/mbed_targets.md

```diff
@@ -148,6 +149,40 @@ The build system errors when you use features outside of this list.
 
 When you use target inheritance, you may alter the values of `features` using `features_add` and `features_remove`. This is similar to the `macros_add` and `macros_remove` mechanism the previous section describes.
 
+#### `config` and `overrides`
+
+_configs_ provide a more flexible way to manage macros for a target. Each configuration has a macro name, as well as a default value and an optional help value. For example, this config flag defines a configuration `lf_clock_src` that defines the source for the low frequency clock on the nRF51 target, with a default value of `NRF_LF_SRC_XTAL`. Compiling sets this value for the macro `MBED_CONF_NORDIC_NRF_LF_CLOCK_SRC`:
```

Comment:
Query: In "a more flexible way", what are we comparing this to? More flexible than what?

Also, could you please clarify the third sentence? When we say "this config flag," are we talking about the example starting on line 156 or something else?

Also, is this example specific to the nRF51, or does it work with other boards, too?

Reply:
I was referencing the section of the style guide below on why modules and boards shouldnt use `macros`. The example is just a config definition I pulled from the NRF51 target, but I'll replace it with something more generic. I should probably move the explanation after the example